examples/custom_outs/expected.cjs

```diff
@@ -1,10 +1 @@
-"use strict";
-Object.defineProperty(exports, "__esModule", {
-    value: true
-});
-exports.a = void 0;
-var a = "a";
-exports.a = a;
-
-
-//# sourceMappingURL=a.cjs.map
\ No newline at end of file
+export var a = "a";
```

Comment:
this change seems incorrect, we output esm modules even though we ran `swc compile --config module.type=commonjs`

Reply:
ah, I see that you're working on it: https://github.com/swc-project/swc/issues/4017#issuecomment-1374141572

I'd rather not hold this release for that, since I've got it all loaded in my brain today. Maybe we just change this example to a TODO.